ECart/Product.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ECart
+{
+    class Product
+    {
+        string Name;
```

Comment:
non public properties should be in camelCase. use name instead.

Reply:
Resolved